Ginger/Ginger/RunSetLib/RunSetOperations.cs

```diff
@@ -19,23 +19,46 @@ public static RunSetConfig CreateNewRunset(string runSetName="", RepositoryFolde
                 if (!InputBoxWindow.GetInputWithValidation(string.Format("Add New {0}", GingerDicser.GetTermResValue(eTermResKey.RunSet)), string.Format("{0} Name:", GingerDicser.GetTermResValue(eTermResKey.RunSet)), ref runSetName, System.IO.Path.GetInvalidPathChars()))
                 {
                     return null;
-                }                    
+                }
+
+                while (CheckAmbiguity(runSetName))
+                {
+                    Reporter.ToUser(eUserMsgKeys.ValueIssue, runSetName + " already exists ! Run Set name should be unique !");
+
```

Comment:
please add dedicated eUserMsgKeys for this message, and not need to use "!" in message, it should explain the issue in more pleseant manner.